### PR TITLE
add the Cooperative Non-Violent Public License

### LINKED ISSUE #430: RFC: choose a license
This is just an issue to try to get a discussion going around which license is best.

We have [one PR open to add the MIT license](https://github.com/datafruits/datafruits/pull/344).
Certainly having a license would be better than having none, but I'd like to explore other possibilities for a license.

Here are a couple different license I've been trying to learn more about.

## Hippocratic License
https://firstdonoharm.dev/

Basically states that the software cannot be used to violate any human rights or cause harm.

Recently was adopted in another project I've contributed to. https://github.com/vcr/vcr/blob/master/LICENSE

## Copyfarleft

I am still learning about this but the idea seems to be that the software cannot be used in any for profit organizations that aren't structured like a cooperative.

https://wiki.p2pfoundation.net/Copyfarleft#Copyfarleft_type_licenses

### PR DESCRIPTION
**Pull request by mcfiredrill**
https://git.pixie.town/thufie/CNPL

resolves #430

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,517 @@
+[Work Title]
```
Comment (mcfiredrill) [inline LICENSE:1]: Oh better actually fill this in...

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+Copyright [Author] [Year]
```
Comment (mcfiredrill) [inline LICENSE:2]:
```suggestion
Copyright The Datafruits Community 2020
```

File: LICENSE (hunk 1 above)
Comment (mcfiredrill) [inline LICENSE:1]:
```suggestion
Datafruits.fm source code
```

**Comment by mcfiredrill** [comment]
I haven't gotten any of the reviewers to approve but I asked about this in the discord and at least a couple people seem to like it so I will merge this.

**Comment by fastdrums** [comment]
i read a good chunk of it, seems good to me?  the cool part about it is
that we can just update/change should something else come along that fits
better

**Review by dudethatbe — APPROVED** [review]

**Comment by mcfiredrill** [comment]
>  can just update/change should something else come along that fits
better

Absolutely, we can always change or update later.

**Review by caass — APPROVED** [review]

**Comment by caass** [comment]
Yeah I dig it

**Pull request merged**